handbook/strategy/recurring-processes.md

```diff
@@ -50,13 +50,13 @@ The goal of this activity is for groups to collaborate by coming up with a strat
 
 ### Project Ideation
 
-Project ideation revolves around the guidelines provided in our [Project Scope guide](/handbook/project-management/scope.md), especially our guidelines about using a "core feature" to help narrrow MVP scope.
+Project ideation revolves around the guidelines provided in our [Project Scope guide](/handbook/project-management/scope.md), especially our guidelines about using a "core feature" to help narrrow MVP scope. The goal of ideation is not just to come up with ideas, but to also form our [project teams](/handbook/manifesto.md#project-teams).
 
 In general, the process is as follows:
 
 1. Prior to kickoff, we will ask for some "core feature" submissions via Google Forms from the entire club
-2. Leads will review the submissions and consolidate them into "ideas", with each lead adopting one idea. These will act as discussion points.
-3. During kickoff, the "ideas" will be presented to the club, and we will form discussion groups based on interest to refine each "idea" back down into a "core feature". Discussions will take place over several iterations.
+2. Leads will review the submissions and consolidate them into "themes", with each lead adopting one idea that they are interested in working on. These will act as topics for discussion groups that will become Launch Pad teams.
```

Comment:
Will leads be locked in this category in the future? Or would leads only lead the discussion groups for the kickoff, and they are free to swap categories afterwards? If it's the former, I think we should add a note saying so - right now it's not super clear to me that the leads would be locked in.

Reply:
No, it is not, as I [mentioned in Slack](https://ubclaunchpad.slack.com/archives/G013BC5RC22/p1601266258064800?thread_ts=1601231416.042300&cid=G013BC5RC22)

Reply:
Leads weren't locked into categories this time either, but we did say pick topics you are interested in, so we might have made a bit of a leap in assumption when moving forward with each lead owning the topic they picked up